source/application/PGM/PGMAppController.cpp

```diff
@@ -186,14 +171,14 @@ void PGMAppController::setupUserInterfaceImplementation()
 void PGMAppController::createPersistentView()
 {
 	PGMPersistentView *pgmPersistentView = new PGMPersistentView;
-	connect( pgmPersistentView, SIGNAL(beamlineStatusSelectedComponentChanged(AMControl*)), this, SLOT(goToBeamlineStatusView(AMControl*)) );
+	connect( pgmPersistentView, SIGNAL(beamlineStatusControlClicked(AMControl*)), this, SLOT(goToBeamlineStatusView(AMControl*)) );
```

Comment:
Was this connect statement not altered in BioXASAppController or was it done in a different pull request?